sensor/common/centralclient/testdata/update-testdata.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# This script will update the testdata for the TLSChallenge tests.
+# Usage: ./update-testdata.sh
+#
+# Prerequisites:
+# - roxcurl
```

Comment:
A pointer to where roxcurl can be found would be great.

Reply:
done